video_xblock/backends/brightcove.py

```diff
@@ -32,28 +32,40 @@ def get_frag(self, **context):
         frag.add_css_url(
             'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'
         )
-        frag.add_javascript(
-            self.render_resource('../static/js/player_state.js', **context)
+        frag.add_content(self.add_js_content(
+            '../static/bower_components/videojs-transcript/dist/videojs-transcript.js'
+        ))
+        frag.add_content(
+            self.add_js_content('../static/js/player_state.js', **context)
         )
-        frag.add_javascript(
-            self.render_resource('../static/js/videojs-tabindex.js', **context)
+        frag.add_content(
+            self.add_js_content('../static/js/videojs-tabindex.js', **context)
         )
-        frag.add_javascript(
-            self.resource_string('../static/js/toggle-button.js')
+        frag.add_content(
+            self.add_js_content('../static/js/toggle-button.js')
         )
-        frag.add_javascript(
-            self.render_resource('../static/js/videojs-transcript.js', **context)
+        frag.add_content(
+            self.add_js_content('../static/js/videojs-transcript.js', **context)
         )
-        frag.add_javascript(self.render_resource(
+        frag.add_content(self.add_js_content(
             '../static/js/videojs_event_plugin.js', **context
         ))
-        frag.add_javascript(self.resource_string(
+        frag.add_content(self.add_js_content(
             '../static/bower_components/videojs-offset/dist/videojs-offset.js'
         ))
-        frag.add_javascript(
-            self.render_resource('../static/js/videojs-speed-handler.js', **context)
+        frag.add_content(
+            self.add_js_content('../static/js/videojs-speed-handler.js', **context)
+        )
+        frag.add_content(
+            self.add_js_content('../static/js/brightcove-videojs-init.js', **context)
         )
         frag.add_css(self.resource_string(
             '../static/css/brightcove.css'
         ))
         return frag
+
+    def add_js_content(self, path, **context):
+        """
+        Helper for adding javascript content to brightcove player
```

Comment:
In this case, it makes sense to avoid using `brightcove` player name in the docstring